plenum/server/consensus/ordering_service.py

```diff
@@ -670,6 +671,7 @@ def last_ordered_3pc(self):
     @last_ordered_3pc.setter
     def last_ordered_3pc(self, lo_tuple):
         self._data.last_ordered_3pc = lo_tuple
+        self.lastPrePrepareSeqNo = lo_tuple[1]
```

Comment:
Do we have (unit) tests that we change `lastPrePrepareSeqNo ` on changing `last_ordered_3pc`?